schema_salad/tests/test_java_codegen.py

```diff
@@ -41,16 +41,6 @@ def test_meta_schema_gen(tmp_path: Path) -> None:
     assert src_dir.exists()
 
 
-def get_data_uri(resource_path: str) -> str:
-    path = get_data(resource_path)
-    assert path
-    return ref_resolver.file_uri(path)
-
-
-cwl_file_uri = get_data_uri("tests/test_schema/CommonWorkflowLanguage.yml")
-metaschema_file_uri = get_data_uri("metaschema/metaschema.yml")
```

Comment:
These schema uris from the Java codegen test file were imported in the Python codegen test file. So I moved them to the `utils.py` module, where it appears we can have code common to other tests.